src/main/java/org/hobbit/core/containerservice/RabbitMQContainerCreator.java

```diff
@@ -0,0 +1,62 @@
+package org.hobbit.core.containerservice;
+
+import org.hobbit.core.Constants;
+import org.hobbit.core.components.AbstractBenchmarkController;
+import org.hobbit.core.components.AbstractCommandReceivingComponent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Implements functionality for RabbitMQ container creation
+ * @author altaf, sourabh, yamini, melisa
+ *
+ */
+public class RabbitMQContainerCreator implements ContainerCreation {
```

Comment:
The "DirectContainerCreator" seems to use RabbitMQ as well. So where is the difference?